bots/john/assets/var/jokes.json

```diff
@@ -1,86 +0,0 @@
-[
-   {
-      "joke":"Did you hear about the guy whose whole left side was cut off? He's all right now."
```

Comment:
Yeah, this is deleted in 240ee77bfc07149e18023e3afcaa168300f1996e.  It's really easy to see if you use `gitk` or another graphical viewer to inspect these commits.